apps/antalmanac/src/lib/enrollmentHistory.ts

```diff
@@ -68,14 +68,13 @@ export class DepartmentEnrollmentHistory {
         this.partialQueryString = DepartmentEnrollmentHistory.QUERY_TEMPLATE.replace('$$DEPARTMENT$$', department);
     }
 
-    async find(courseNumber: string): Promise<EnrollmentHistory | null> {
+    async find(courseNumber: string): Promise<EnrollmentHistory[] | null> {
         const cacheKey = this.department + courseNumber;
```

Comment:
Optional, but this might be useful as a static method, so that creating and decoding cache keys are centralized. 
This is pretty insubstantial in this scenario tbh, but might be useful thinking in some other case.
e.g.
```tsx
// Example: splitting with '/'
createKey(courseNumber: string) {
  return this.department + '/' + courseNumber;
}

// For example, this behavior could be enabled
decodeKey(key: string) {
  const [department, courseNumber] = key.split('/')
}
```